README.md

```diff
@@ -18,12 +38,33 @@ example-pkg-upload:
       id: test-action
       uses: chassyflow/actions-upload
       with:
+        name: 'example-image'
         architecture: 'ARM64'
         os: 'ubuntu'
         os_version: '22.04'
+        raw_disk_scheme: 'IMG'
         type: 'IMAGE'
         path: '**/release.img'
         classification: 'RFSIMAGE'
+example-archive-upload:
+  name: Example Archive Upload
+  runs-on: ubuntu-latest
+  env: CHASSY_TOKEN
+  steps:
+    - name: Checkout
+      id: checkout
+      uses: actions/checkout@v4
+    - name: Upload Archive to Chassy
+      id: test-action
+      uses: chassyflow/actions-upload
+      with:
+        name: 'example-archive'
+        architecture: 'ARM64'
+        os: 'ubuntu'
+        os_version: '22.04'
+        type: 'IMAGE'
+        path: '**/bundle.zip'
+        entrypoint: "entrypoint.sh"
```

Comment:
entrypoint is going to be a list.

- entrypoint.sh
- optionalarg1
- optionalarg2
- optionalargX